src/main/java/nl/novi/behuiswerkles11/controllers/TelevisionsController.java

```diff
@@ -1,39 +1,49 @@
 package nl.novi.behuiswerkles11.controllers;
 
-import org.springframework.http.HttpStatus;
-import org.springframework.http.HttpStatusCode;
+import nl.novi.behuiswerkles11.models.Television;
+import nl.novi.behuiswerkles11.repositories.TelevisionRepository;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+import java.util.Optional;
+
 @RestController
 @RequestMapping("televisions")
 public class TelevisionsController {
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionsController(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
+
     @GetMapping()
-    public ResponseEntity<String> getTelevisions() {
-//        return ResponseEntity.ok("televisions");
-        return new ResponseEntity<>("all televisions", HttpStatus.OK);
+    public ResponseEntity<List<Television>> getTelevisions() {
+        return ResponseEntity.ok(televisionRepository.findAll());
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<String> getOneTelevision(@PathVariable Integer id) {
-        return ResponseEntity.ok("televisionId: " + id);
+    public ResponseEntity<Optional<Television>> getOneTelevision(@PathVariable Long id) {
```

Comment:
Het is beter om niet een Optional te returnen naar de client, maar de Television (beter nog een Dto, maar dat hoeft in deze opdracht nog niet) door de get-methode van Optional aan te roepen: `return ResponseEntity.ok(televisionRepository.findById(id).get());`